btree/btree.c

```diff
@@ -11091,6 +10788,57 @@ static void ut_mt_tree_oper(void)
 	btree_ut_num_threads_tree_oper(0);
 }
 
+static bool validate_nodes_on_be_segment(struct segaddr *rnode_segaddr)
+{
+	const struct node_type     *nt;
+	struct node_data {
+		struct segaddr      node_segaddr;
+		uint16_t            rec_idx;
+		}                   stack[MAX_TREE_HEIGHT + 1];
+	uint32_t                    stack_level = 0;
+	struct nd                   _node;
+	struct slot                 _slot = { .s_node = &_node };
```

Comment:
Please use other name for _node and _slot, which do not start from _ .

Reply:
done.